frontend/src/app/app.component.ts

```diff
@@ -10,10 +11,16 @@ export class AppComponent implements OnInit {
   headerHeight = 70;
   contentHeight: number;
   footerHeight = 40;
+  logoSrc: string;
+  apptitle: string;
 
   @ViewChild('content', {static: true}) content: ElementRef;
 
-  constructor(private renderer: Renderer2) {
+  constructor(private renderer: Renderer2, private applicationService: ConfigService) {
```

Comment:
Maybe not in this story, but we should get rid of this Renderer2...

Reply:
ok i'll do it in the next story